src/jit/compiler.cpp

```diff
@@ -98,10 +98,15 @@ void Compiler::JitLogEE(unsigned level, const char* fmt, ...)
     if (verbose)
     {
         logf_stdout(fmt, args);
+        // logf_stdout can modify args so we need to reset it
+        va_end(args);
```

Comment:
This would look better:

#ifndef CROSSGEN_COMPILE
    if (verbose)
    {
        va_start(args, fmt);
        logf_stdout(fmt, args);
        va_end(args);
    }
#endif

    va_start(args, fmt);
    vlogf(level, fmt, args);
    va_end(args);